types/src/ledger_info.rs

```diff
@@ -199,6 +199,17 @@ impl Display for LedgerInfoWithV0 {
     }
 }
 
+impl Debug for LedgerInfoWithV0 {
+    fn fmt(&self, f: &mut Formatter) -> std::fmt::Result {
+        write!(
+            f,
+            "LedgerInfo {{ commit_info: BlockInfo {{ epoch: {:?}, version: {:?} }} }}",
```

Comment:
Not sure if I need all the labels like `commit info: BlockInfo`, but I put it there to follow the default way `Debug` is printed out. If you want it to be more compact, please let me know.

Reply:
My only concern with this approach is that others might still want the `Debug` implementation for `LedgerInfoWithV0` to show all the fields. Perhaps there's a reasonable middle ground here:

Implement `Display` for `LedgerInfoWithV0` (i.e., this implementation block), and then when we implement `Debug` for `AdvertisedData` we call the `Display` implementation for `LedgerInfoWithV0`. Thoughts?

Reply:
Yeah that was my concern initially too (that's why I kept the other way around to hear your opinions).

The middle ground way seems reasonable to me. However, if we still have `impl Debug AdvertisedData`, should we just output what we want with `LedgerInfoWithV0` inside `impl Debug AdvertisedData` and leave `LedgerInfoWithV0` untouched (i.e., keep its original version before I changed)?

Reply:
Yeah, I see what you're saying. Personally, I like this approach because it means that if we ever need to display a `LedgerInfoWithSignatures` in the code again (which, actually tends to occur a lot during debugging), we can now use the `Display` method and we won't have to reimplement it.